assets/js/blocks/price-filter/edit.js

```diff
@@ -166,7 +173,7 @@ export default function( { attributes, setAttributes } ) {
 					<Disabled>
 						<Block attributes={ attributes } isPreview />
```

Comment:
Is it okay to be passing through the `attributes.className` here (I'm guessing yes, but just checking)? The question applies to other blocks as well.

Reply:
It's ok because we are not using it inside `<Block>`, alternatively we could pass the attributes without the `className`, but I think there is no need to remove it if we make sure not to print it again.